.devcontainer/populate_keycloak.sh

```
@@ -0,0 +1,224 @@
+#!/bin/bash
+# Populates Keycloak with the realm "empire" and users
+# and all settings for CouchDB to trust the IdP's certs
+
+# Wait until CouchDB answers
+until curl -fsS "http://localhost:5984/_up" >/dev/null 2>&1; do sleep 2; done
+# Wait until Keycloak answers
+until curl -fsS "http://localhost:8090/realms/master" >/dev/null 2>&1; do sleep 2; done
+
+# Variables needed
+now=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck (0.10.0)

[warning] now appears unused. Verify use (or export if used externally).

(SC2034)